src/ServiceManagement/Azure.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '5.3.0'
+ModuleVersion = '5.3.1'
```

Comment:
Please confirm it with the team, but your added cmdlets may qualify a higher level of version update, e.g. ```5.4.0``` or even the 1st digit.

Reply:
Please do not increase the module version numbers. We handle version number incrementation when we create releases.